examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -191,18 +190,18 @@ genesisinfo {"info": "02c37240d51ea3a03278f3ec3935daa1f59724245eb8af3af4528ed77a
+    "powData": "c149e35e030f0f20c149e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20c049e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20bf49e35e030f0f20be49e35e030f0f20be49e35e040f0f20be49e35e040f0f20be49e35e040f0f20be49e35e040f0f20be49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bd49e35e040f0f20bc49e35e040f0f20bc49e35e040f0f20", 
     "mcBlockHeight": 220, 
-    "mcNetwork": "regtest"
+    "mcNetwork": "regtest", 
+    "withdrawalEpochLength": 10 
 }
 ```
 
 **Step 11: Update Sidechain configuration file**
```

Comment:
"missing zencliCommandLine = "/home/ponta/zend_oo/src/zen-cli" in websocket part of config", please add information about zencliCommandLine  as well